data/json/items/gunmod/stock.json

```diff
@@ -156,6 +156,22 @@
     "handling_modifier": -10,
     "flags": [ "COLLAPSED_STOCK", "IRREMOVABLE" ]
   },
+  {
+    "id": "stock_none",
+    "type": "GUNMOD",
+    "name": { "str": "sawn-off stock" },
+    "description": "A simple saw job cutting the original stock off this weapon.  Makes it much harder to handle but much smaller.",
+    "install_time": "5 m",
+    "material": [ "wood" ],
+    "symbol": ":",
+    "color": "dark_gray",
+    "location": "stock mount",
+    "handling_modifier": -10,
+    "mod_targets": [ "smg", "rifle", "shotgun" ],
+    "min_skills": [ [ "weapon", 1 ] ],
```

Comment:
I don't think "a simple saw job" requires any skill to do🤔
```suggestion
```

Reply:
I think skill might be mandatory, the "no stock" item never drops so this line doesn't actually matter.

Sawing it doesn't require this skill it forcibly adds the mod.

Reply:
Skill is not a mandatory field, shortened barrel doesn't have it.